suite/tests/client-interface/process-id.dll.c

```diff
@@ -0,0 +1,14 @@
+#include "dr_api.h"
```

Comment:
Please include the license + copyright header

Reply:
Done.